### ISSUE TITLE
Error  Module `AccessibilityInfo` does not exist in the Haste module map while running app.

### ISSUE THREAD

**Issue by sandeepsinghs** [mmazzarolo/react-native-modal-datetime-picker#206]
While running app I am getting error as 
Unable to resolve module `AccessibilityInfo` from ....  Module `AccessibilityInfo` does not exist in the Haste module map
 that's why I am not able to run app.

Please help.

**Comment by mmazzarolo** [comment]
Hi! Please follow the issue template before submitting a new issue, thank you!
